<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Шахматы :: Главное Меню</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="sizeLabel">
      <property name="text">
       <string>Размер доски (N)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="boardSizeSpinBox">
      <property name="minimum">
       <number>4</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="requiredLabel">
      <property name="text">
       <string>Кол-во требуемых фигур (L)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="requiredFiguresSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="placedLabel">
      <property name="text">
       <string>Кол-во размещённых фигур (K)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="placedFiguresSpinBox">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="createBoardButton">
        <property name="text">
         <string>Создать доску</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="drawBoardButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Нарисовать доску</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
